microsoft-365/security/mtp/advanced-hunting-overview.md

```diff
@@ -50,6 +50,19 @@ This capability is similar to [advanced hunting in Microsoft Defender for Endpoi
 
 To use advanced hunting, [turn on Microsoft 365 Defender](mtp-enable.md).
 
+### Before you begin
+
+Users need one of the following levels of permissions to access Microsoft Defender:
+
+- Full access (read and write)
+- Read-only access
+
+**Full access**:
+Users with full access can save, modify, and share a query. Assigning full access rights requires adding the users to the "Security Administrator" or "Global Administrator" AAD built-in roles.
```

Comment:
kindly replace "AAD built-in roles." with:

"built-in roles in Azure Active Directory (AAD)."